TCC.Lib/Dependencies/ExternalDependecies.cs

```diff
@@ -3,14 +3,15 @@
 using System.IO.Compression;
 using System.Net;
 using System.Threading.Tasks;
+using TCC.Lib.Helpers;
 
 namespace TCC.Lib.Dependencies
 {
     public class ExternalDependecies
     {
         private const string ExeTar = @"Libs\tar.exe";
-        private const string ExeOpenSsl = @"Libs\openssl.exe";
-        private const string CnfOpenSsl = @"Libs\openssl.cnf";
+        //private const string ExeOpenSsl = @"Libs\openssl.exe";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)